tests/unit/fetcher-test.js

```diff
@@ -372,7 +372,7 @@ describe('Fetcher', () => {
     it('should set the Accept header', () => {
       options = fetcher.initFetchOptions(uri, options)
 
-      let expectedHeader = 'image/*;q=0.9, */*;q=0.1, application/rdf+xml;q=0.9, application/xhtml+xml, text/xml;q=0.5, application/xml;q=0.5, text/html;q=0.9, text/plain;q=0.5, text/n3;q=1.0, text/turtle;q=1'
+      let expectedHeader = 'image/*;q=0.9, */*;q=0.1, application/rdf+xml;q=0.9, application/xhtml+xml, text/xml;q=0.5, application/xml;q=0.5, text/html;q=0.9, text/plain;q=0.5, text/n3;q=1.0, text/turtle;q=1, application/ld+json;q=0.4'
```

Comment:
I think should be `application/ld+json;q=0.9` as, given a JSDON/LD file we parse it completely with no data loss.  The only reason we don't put it as 1.0 is for systems which can provide any equally well we prefer turtle as it tends to be smaller, simpler, and should be faster to parse.